content/miscellaneous/self_contained_mode.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Self-Contained Mode"
+#menuTitle: ""
+chapter: false
+weight: 10
+---
+
+## Self-Contained Mode
+
+### Traditional Detection Mode (deprecated)
+
+The default mode in CRS 3.x is Anomaly Scoring mode, you can verify this
+is your mode by checking that the SecDefaultAction line in the
+csr-setup.conf file usees the pass action:
+
+```apache
+SecDefaultAction "phase:2,pass,log"
+```
+
+{{% notice warning %}}
+From version 3.0 onwards, Anomaly Scoring is the default detection mode. Traditional detection mode is discouraged.
+{{% /notice %}}
+
+{{% notice warning %}}
```

Comment:
I think this 2nd warning is not necessary. This mode is easy enough to understand and I do not think anything can go wrong really by misusing it. Our default installation with threshold 5 is very much this mode already.

Reply:
Removed the second warning block.

I'll have a proper look at the `self_contained_mode.md` content in the future, but for now I just need to move it out of the anomaly scoring "key concept" page (it was part of the original content on that page).